src/Algebra/Structures.agda

```diff
@@ -696,7 +714,7 @@ record IsNonAssociativeRing (+ * : Op₂ A) (-_ : Op₁ A) (0# 1# : A) : Set (a
   field
     +-isAbelianGroup : IsAbelianGroup + 0# -_
     *-cong           : Congruent₂ *
-    identity         : Identity 1# *
+    *-identity       : Identity 1# *
```

Comment:
What version was `IsNonAssociativeRing` added? Is this is a breaking change? We should also change `zero` if we're changing this right?

Reply:
Do you mean _after_ v1.7.1? Surely it wasn't added _between_ v1.7 and v1.7.1 ....

Reply:
It was added in #1795 This was in August